_security-plugin/configuration/ldap.md

```diff
@@ -537,3 +537,106 @@ authz:
           - 'cn=Jane Doe,ou*people,o=TEST'
           - '/\S*/'
 ```
+
+### (Advanced) Configuring multiple user and role bases
+
+If you need to configure multiple user bases in authc and/or authz section, this can be done using below syntax:
+
+```yml
+        ...
+        bind_dn: cn=admin,dc=example,dc=com
+        password: password
+        users:
+          primary-userbase:
+             base: 'ou=people,dc=example,dc=com'
+             search: '(uid={0})'
+          secondary-userbase:
+             base: 'cn=users,dc=example,dc=com'
+             search: '(uid={0})'
+        username_attribute: uid
+        ...
+```
+
+Similarly, multiple role bases in authz section can be configured with:
```

Comment:
Similarly, use the following setup to configure multiple role bases in the authz section:

Reply:
Done